trace/cloud-trace-demo-app/README.md

```diff
@@ -0,0 +1,45 @@
+# cloud-trace-demo-app
+
+Public Docker image: gcr.io/cloud-trace-demo/demo-image:latest
+
+Open this demo app in [Google Cloud Shell](https://cloud.google.com/shell/docs/). This includes necessary tools.
+
+[![Open Cloud Trace Demo APP in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=trace/cloud-trace-demo-app/README.md&amp;cloudshell_tutorial=README.md)
+
+Run `gcloud init` to set up your region and zone.
+
+#### Demo Requirements
+If you are using cloud shell, skip to the next section.
+
+1. Install gcloud https://cloud.google.com/sdk/install
+2. Install kubectl https://kubernetes.io/docs/tasks/tools/install-kubectl/
+3. Install docker https://docs.docker.com/install/
+
+#### Google Container Registry Image Setup
+If you are using the provided image, skip to the next section.
+4. Get default project id:
+    `PROJECT_ID=$(gcloud config get-value project)`
+5. Build Image:
+    `sudo docker build -t gcr.io/${PROJECT-ID}/cloud-trace-demo .`
+6. Upload Image to Container Registry:
+    `sudo gcloud docker -- push gcr.io/${PROJECT-ID}/cloud-trace-demo-test:v1`
+
+#### Create a GKE cluster
+7. Enable Google cloud and set up regions
+    `gcloud init`
+8. Enable GKE api & billing:
+    `gcloud services enable container.googleapis.com`
+9. Create a GKE cluster named "demo":
+    `gcloud container clusters create demo`
+
+#### Send Requests to See Generated Traces
+
+10. Run setup.sh to apply the YAML files. Note this file configures the default image we provide.
+    If you are using your own image, please go and change the image string in
+    the YAML file and templates.
+    `./setup.sh`
+11. Send request to the last service:
+    `curl $(kubectl get svc cloud-trace-demo-a -ojsonpath='{.status.loadBalancer.ingress[0].ip}')`
+12. Visit [Trace List](https://pantheon.corp.google.com/traces/list) to check traces generated
```

Comment:
Consider adding a screenshot that shows what your trace list should look like if you're successful

Reply:
@dukeng I remember you have a project up and running with our demo app. Do you mind take a screenshot and add the image file here? Thanks!

Reply:
screenshot would still be good here

Reply:
Done. Screenshot added.